src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/blob.py

```diff
@@ -170,7 +170,7 @@ def storage_blob_download_batch(client, source, destination, source_container_na
 def storage_blob_upload_batch(client, source, destination, pattern=None, source_files=None,
                               destination_container_name=None, blob_type=None,
                               content_settings=None, metadata=None, validate_content=False,
-                              maxsize_condition=None, max_connections=1, lease_id=None,
+                              maxsize_condition=None, max_connections=2, lease_id=None,
```

Comment:
2 was the original default. Troy made it 1 to address my comment on this issue for his batch PR only, so I changed it back to 2 and implement the workaround for both commands in a single place because the default should be 2 when the SDK is fixed.